### PR TITLE
Adding Block Rewards

### PR DESCRIPTION
**Pull request by Rhanok09**
Submitted the Block rewards proposal.

### CODE REVIEW

File: cps/CPXX.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+**Author(s):** Governance and Coordination Group (ImdioR, Orhan)
```
Comment (lucasvo) [inline cps/CPXX.md:1]: I would add a title here:

`# CP6: Implement a block reward for Centrifuge Parachain`

File: cps/CPXX.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+**Technical/non-technical proposal:** Technical
+
+**Date proposed:** 2022-11-16
+
+## Short Summary
+Implement a block reward to Centrifuge Parachain
+Improvement of Collator Reward Cycle
+High-level objective
+This proposal is aimed to improve Collators management and implement a block reward to the Centrifuge Parachain and funding the Centrifuge Treasury, which will solve a number of issues.
+Background
```
Comment (lucasvo) [inline cps/CPXX.md:14]:
```suggestion
### Background

```

File: cps/CPXX.md (hunk 1 above)
Comment (lucasvo) [reply cps/CPXX.md:1]: You should rename this file to CP6.md you can do this in the editor

**Review by lucasvo — APPROVED** [review]

File: cps/CP0006.md [hunk 3]
```diff
@@ -1,9 +1,10 @@
 ---
-Title: CP-0006: Add a block reward and improve Collator cycle
-Author(s): Governance and Coordination Group (ImdioR, Orhan)
-Advisor(s): Cassidy, Miguel
-Technical/non-technical proposal: Technical
-Date proposed: 2022-11-16
+title: CP-0006: Add a block reward and improve Collator cycle
+authors: Governance and Coordination Group (ImdioR, Orhan)
+advisors: Cassidy, Miguel
+proposal_type: CP-2
```
Comment (Rhanok09) [inline cps/CP0006.md:5]: This is a CP-4 (technical). A CP-2 is a request for funding.

**Pull request merged**